main.go

```diff
@@ -73,12 +191,6 @@ func main() {
 	}
 
 	client := makeClient()
-	sc, err := stan.Connect(clusterID, clientID, stan.NatsURL("nats://"+config.NatsAddress+":4222"))
-	if err != nil {
-		log.Fatalf("Can't connect to %s: %v\n", "nats://"+config.NatsAddress+":4222", err)
-	}
-
-	startOpt := stan.StartWithLastReceived()
 
 	i := 0
 	mcb := func(msg *stan.Msg) {
```

Comment:
Since we dropped mcb for `messageHandler` somewhere else, let's rename it to `messageHandler` here also.

Reply:
Done